client/src/app/domain/models/poll/poll-constants.ts

```diff
@@ -164,6 +165,7 @@ export const PollPercentBaseVerbose = {
     valid: _(`All valid ballots`),
     cast: _(`All casted ballots`),
     entitled: _(`All entitled users`),
+    entitled_present: _(`All present entitled users`),
```

Comment:
You probably also need to implement the new base, but I'm not sure where this is located. Maybe @luisa-beerboom can help you here

Reply:
Yes, I didn't see that. I work on that.

Reply:
Done.